### PR TITLE
[Feature] Postmark / Mailgun - Email providers

### PR DESCRIPTION
**Pull request by Civolilah**
Here is the screenshot of implemented solution for email providers (Postmark and Mailgun):

Screenshot 2023-02-20 at 17 34 13

@beganovich @turbo124 I just noticed one thing that we don't have `gmail` as a translated keyword, but `microsoft` was in the translation files. Should we have `gmail` and `microsoft` as translated keywords or can we put it as `plain text`. 

Also, David mentioned in the ticket that this should only be possible for Pro/Enterprise plans. The entire page (Email Settings) is available for saving only if the user has one of those plans. That covers that part.

Let me know your thoughts.

### CODE REVIEW

File: src/pages/settings/email-settings/EmailSettings.tsx [hunk 1]
```diff
@@ -94,6 +94,56 @@ export function EmailSettings() {
...
+              handleChange('settings.email_sending_method', value)
+            }
+          >
+            <option defaultChecked value="default">
+              {t('default')}
+            </option>
+            <option value="gmail">{t('gmail')}</option>
+            <option value="microsoft">{t('microsoft')}</option>
+            <option value="client_postmark">{t('postmark')}</option>
+            <option value="client_mailgun">{t('mailgun')}</option>
```
Comment (beganovich) [inline src/pages/settings/email-settings/EmailSettings.tsx:110]: No need for translations for company names. They are same on all languges.

**Review by beganovich — CHANGES_REQUESTED** [review]
Looks good, small change needed 👍

File: src/pages/settings/email-settings/EmailSettings.tsx (hunk 1 above)
Comment (Civolilah) [reply src/pages/settings/email-settings/EmailSettings.tsx:110]: Changed

**Review by beganovich — APPROVED** [review]
Looks good, thanks :+1:

**Pull request merged**